conf/modules.d/p0f.conf

```diff
@@ -0,0 +1,46 @@
+# Please don't modify this file as your changes might be overwritten with
+# the next update.
+#
+# You can modify '$LOCAL_CONFDIR/rspamd.conf.local.override' to redefine
+# parameters defined on the top level
+#
+# You can modify '$LOCAL_CONFDIR/rspamd.conf.local' to add
+# parameters defined on the top level
+#
+# For specific modules or configuration you can also modify
+# '$LOCAL_CONFDIR/local.d/file.conf' - to add your options or rewrite defaults
+# '$LOCAL_CONFDIR/override.d/file.conf' - to override the defaults
+#
+# See https://rspamd.com/doc/tutorials/writing_rules.html for details
+
+p0f {
+  # Disable module by default
+  enabled = false;
+
+  # Path to the unix socket that p0f listens on
+  socket = '/tmp/p0f.sock';
```

Comment:
@denpamusic Would you explain the reason of using `/tmp` for unix sockets? Wouldn't it be better to use `/var/run/p0f.sock` instead?

Reply:
Sorry, I've come from FreeBSD-land where most sockets created in /tmp by default as /var/run in only writable by root, so it's kinda habit of mine... You're welcome to change it of course, if you think that I'll be better suited for most users. This PR is already merged, so you'll need to create new one.

Reply:
After doing a bit of research I've decided to change default socket location to `/var/run/p0f.sock` as you've recommended. I've submitted #3043 that includes this change as well as fixes a couple other problems I've noticed during plugin usage.

Reply:
Are you running p0f as an unprivileged user? How? AFAIU it needs permission to read from `BPF`.

Reply:
You're right, but p0f can deal with it with `-u user` flag. Basically it's opens bpf as root and the drops privileges.